components/authorization_simpler/app/decorators/models/user_decorator.rb

```
@@ -0,0 +1,28 @@
+User.class_eval do
+  # KT: The 'User#roles' attribute is a special Rails Postgres 'array' attribute type ...
+  # http://blog.arkency.com/2014/10/how-to-start-using-arrays-in-rails-with-postgresql/
+
+  # KT TODO: what to do about the scope :admin in core -> user.rb
+  scope :admins, -> { where("'admin' = ANY (roles) OR admin = true") }
+
+  # NB: Don't modify the 'roles' array directly, use these methods ...
+  # http://stackoverflow.com/questions/17283854/new-data-not-persisting-to-rails-array-column-on-postgres
+  def add_role(role)
```

Worth noting -- if you try to do this:

`user.roles << 'admin'` OR `user.roles.push 'admin'`

... after saving, when you reload the record, the changes will not have taken.  It fails silently.  Details in the post I linked to.